### ISSUE TITLE
mappings do not work in batch target

### ISSUE THREAD

**Issue by rehlers** [mkloubert/vscode-deploy-reloaded#2]
### Description

I have directory mappings configured for my sftp targets. They work correctly when I deploy the file directly to each server. When I include each target as part of a batch target, it seems to be ignoring the mappings.

### Actual behavior

type batch target ignores mappings from each target

### Expected behavior

type batch target should use the mappings from each target

### Steps to reproduce

Deploy a file that uses a mapping from an individual server to see that it works.
Then use a batch target type that includes the server to see that the mapping is ignored.

#### Example config

```json
{
    "deploy.reloaded": {
        "targets": [
            {
                "type": "sftp",
                "name": "server01",
                "host": "server01", "port": 22,
                "user": "uname", "password": "pword",
                "mappings": 
                    {
                        "/pp/**/*": "/usr/local/etc/pp",
                        "/misc/**/*": "/usr/local/etc"
                    }
            },
            {
                "type": "sftp",
                "name": "server02",
                "host": "server02", "port": 22,
                "user": "uname", "password": "pword",
                "mappings": 
                    {
                        "/pp/**/*": "/usr/local/etc/pp",
                        "/misc/**/*": "/usr/local/etc"
                    }
            },
            {
                "type": "batch",
                "name": "All Servers",
                "targets": ["server01", "server02"]
            }

        ]
    }
}
```

### Screenshot

{Please write here, if possible}

### Your environment

- Operating system: MacOS 10.12.6
- Visual Studio Code version: 1.19.1
- Extension version: 13.0

### Additional comments

{Please write here, if there is something more to tell}

**Comment by mkloubert** [comment]
@rehlers 

**The issue should be fixed since 0.14.2.**

Can you check if all targets still work with the new version as expected?

**Comment by rehlers** [comment]
working great now, thank you so much for the fast response!
